src/bulk_insert/bulk_insert.c

```diff
@@ -17,7 +18,8 @@ typedef enum {
 	BI_BOOL,
 	BI_DOUBLE,
 	BI_STRING,
-	BI_LONG
+	BI_LONG,
+	BI_ARRAY,
```

Comment:
Let's explicitly number the values
```
BI_NULL = 0,
BI_BOOL = 1,
...
```

That way we can guarantee no confusion between the server and the client.